src/iocore/cache/RamCacheLRU.cc

```diff
@@ -47,7 +48,7 @@ struct RamCacheLRU : public RamCache {
   void init(int64_t max_bytes, Vol *vol) override;
 
   // private
-  uint16_t *seen = nullptr;
+  std::vector<bool> *seen = nullptr;
```

Comment:
Are you sure you don't want `std::unique_ptr<std::vector<bool>> seen;`

Reply:
I decided not to rewrite more than was necessary.